Fix:

warning: result of comparison of constant 0 with expression of type 'bool' is always false [-Wtautological-constant-compare]

https://sourceforge.net/p/avarice/bugs/26/
--- src/jtagprog.cc.orig	2020-07-09 16:45:12.000000000 -0500
+++ src/jtagprog.cc	2020-07-09 16:49:18.000000000 -0500
@@ -207,7 +207,7 @@
 void jtag1::enableProgramming(void)
 {
     programmingEnabled = true;
-    if (doSimpleJtagCommand(0xa3, 1) < 0)
+    if (!doSimpleJtagCommand(0xa3, 1))
     {
         fprintf(stderr, "JTAG ICE: Failed to enable programming\n");
         throw jtag_exception();
@@ -218,7 +218,7 @@
 void jtag1::disableProgramming(void)
 {
     programmingEnabled = false;
-    if (doSimpleJtagCommand(0xa4, 1) < 0)
+    if (!doSimpleJtagCommand(0xa4, 1))
     {
         fprintf(stderr, "JTAG ICE: Failed to disable programming\n");
         throw jtag_exception();
@@ -230,7 +230,7 @@
 // (unless the save-eeprom fuse is set).
 void jtag1::eraseProgramMemory(void)
 {
-    if (doSimpleJtagCommand(0xa5, 1) < 0)
+    if (!doSimpleJtagCommand(0xa5, 1))
     {
         fprintf(stderr, "JTAG ICE: Failed to erase program memory\n");
         throw jtag_exception();
